packages/expo-ads-admob/src/AdMob.ts

```diff
@@ -0,0 +1,23 @@
+import { UnavailabilityError } from '@unimodules/core';
+import ExpoAdsAdMob from './ExpoAdsAdMob';
+import ExpoAdsAdMobInterstitial from './ExpoAdsAdMobInterstitialManager';
+import ExpoAdsAdMobRewarded from './ExpoAdsAdMobRewardedVideoAdManager';
+
+let _testDeviceID: string | null = null;
+export function _getTestDeviceID(): string | null {
+  return _testDeviceID;
+}
+
+export async function setTestDeviceID(testDeviceID: string | null): Promise<void> {
+  _testDeviceID = testDeviceID;
+  if (ExpoAdsAdMob.setTestDeviceID) {
+    return await ExpoAdsAdMob.setTestDeviceID(testDeviceID || '');
```

Comment:
```suggestion
    await ExpoAdsAdMob.setTestDeviceID(testDeviceID || '');
```
In line 16 we don't return the `Promise.all`, if we remove `return` result of awaiting this Promise will always be the same -> void.

Suggestion only valid if we decide not to upgrade Android in this same PR.